packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -32,7 +32,7 @@ processors:
           value: 'failed to parse time field ({{{ _temp.ts_nano }}}): {{{ _ingest.on_failure_message }}}'
 - pipeline:
     name: '{{ IngestPipeline "flows" }}'
-    if: ctx.cisco_meraki.event_type == 'flows'
+    if: ctx.cisco_meraki.event_type == 'flows' || ctx.cisco_meraki.event_type == 'firewall' || ctx.cisco_meraki.event_type == 'vpn_firewall' || ctx.cisco_meraki.event_type == 'cellular_firewall' || ctx.cisco_meraki.event_type == 'bridge_anyconnect_client_vpn_firewall'
```

```suggestion
    if: ['flows', 'firewall', 'vpn_firewall', 'cellular_firewall', 'bridge_anyconnect_client_vpn_firewall'].contains(ctx.cisco_meraki.event_type)
```

---

Sorry this should be 
```suggestion
    if: "['flows', 'firewall', 'vpn_firewall', 'cellular_firewall', 'bridge_anyconnect_client_vpn_firewall'].contains(ctx.cisco_meraki.event_type)"
```